pkg/component/kubernetes/apiserver/apiserver_test.go

```diff
@@ -1274,6 +1288,64 @@ resources:
 					Expect(kapi.Deploy(ctx)).To(MatchError(ContainSubstring("either the name of an existing secret or both certificate and private key must be provided for TLS SNI config")))
 				})
 			})
+			It("should create the istio tls secrets for feature gate IstioTLSTermination", func() {
```

Comment:
Please leave a newline between the tests.

```suggestion

			It("should create the istio tls secrets for feature gate IstioTLSTermination", func() {
```